### ISSUE TITLE
"stack bench" fails

### ISSUE THREAD

**Issue by robx** [liqd/thentos#261]
```
$ stack bench
While constructing the BuildPlan the following exceptions were encountered:

--  While attempting to add dependency,
    Could not find package pronk in known packages

--  Failure when adding dependencies:    
      pronk: needed (>=0.1.1), but not present in build plan
    needed for package: thentos-tests-0.0.1
```

This is how far I got, which gave me pages of dependency errors:

```
diff --git a/stack.yaml b/stack.yaml
index 029c1b1..770982d 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -4,6 +4,8 @@ packages:
 - 'thentos-tests'
 - 'thentos-adhocracy'
 - 'services/helloworld'
+- location: 'submodules/pronk'
+  extra-dep: true
 - location: 'submodules/servant/servant'
   extra-dep: true
 - location: 'submodules/servant/servant-server'
@@ -17,13 +19,16 @@ packages:
 extra-deps:
 - acid-state-0.12.4
 - configifier-0.0.5
+- criterion-0.6.2.1
 - cryptonite-0.5
 - digestive-functors-0.8.0.0
 - digestive-functors-blaze-0.6.0.6
 - digestive-functors-snap-0.6.1.3
 - email-validate-2.1.3
 - functor-infix-0.0.3
+- hastache-0.5.1
 - lio-0.11.5.0
+- meldable-heap-2.0.3
 - regex-easy-0.1.0.0
 - regex-pcre-0.94.4
 - sandi-0.3.5
@@ -31,6 +36,8 @@ extra-deps:
 - snap-blaze-0.2.1.3
 - snap-loader-static-0.9.0.2
 - snaplet-acid-state-0.2.7
+- statistics-0.10.5.2
+- text-1.2.1.1
 - uri-bytestring-0.1.6
 
 resolver: lts-2.18
```

**Comment by jkarni** [comment]
Doing the same thing from `pronk` that we do for `servant` should fix it.

**Comment by fisx** [comment]
@robx: i think this is fixed in #301.  if not, please re-open.
